tools/packaging/packaging.psm1

```diff
@@ -243,7 +243,7 @@ function Start-PSPackage {
                     AssetsPath = "$PSScriptRoot\..\..\assets"
                     LicenseFilePath = "$PSScriptRoot\..\..\assets\license.rtf"
                     # Product Guid needs to be unique for every PowerShell version to allow SxS install
```

Comment:
This comment should probably be changed to: Product Code needs to be different for every version